INSTALL.rst

```diff
@@ -1,6 +1,244 @@
+
+.. code-block:: python
+
+    import sys
+    print(sys.executable)
+    import skimage
+    print("scikit-image installed with version " + skimage.__version__)
+
+to make sure the executable used matches your expectations.
+
+**sudo and pip**
+
+Never use ``sudo`` when installing packages with ``pip``. This
+may cause ``pip`` to overwrite important files for your operating system
+which may require you to completely reinstall your system.
+
+
+**User local packages**
+
+Packages installed by your user account (using pip) may
+take precedence over packages installed in your python environment.
+If you have trouble getting the correct version of ``scikit-image``
+take a look at the contents of the directories (if they exist):
+
+- ``~/.local/lib/python3.6``
+- ``~/.local/lib/python3.7``
+- ``~/.local/lib/python3.8``
+
+If you are using virtual environments, conda, or the system package manager,
+then the installation files in those directories are not necessary, and can
+safely be deleted.
+
+
+Supported platforms
+-------------------
+
+The platforms officially supported by scikit-image are:
+
+1. Windows 64 bit
+2. Mac OSX
+3. Linux 64 bit
+
+Since Windows 32 bit remains a popular platform, we continue to support it for
+the time being.  However, we strongly recommend all users of Windows 32 bit
+begin to switch over to Windows 64 bit if they can.
+
+We are very insterested in learning how ``scikit-image`` is
+`used <https://github.com/scikit-image/scikit-image/issues/4375>`_.
+Help us learn what other platforms you would like to install scikit-image!
+
+Unsupported platforms include:
+
+1. Linux on 32 bit Intel processors.
+2. Linux on 32 bit arm (Raspberry Pi running Rapsbian): While we do not official support
+   this distribution, we point users to `piwheels <https://wwww.piwheels.org>`_
+   and
+   `scikit-image's specific page <https://www.piwheels.org/project/scikit-image/>`_.
+
+   - You may need to install additional system dependencies listed for
+     `imagecodecs <https://www.piwheels.org/project/imagecodecs/>`_.
+     See
+     `issue 4721 <https://github.com/scikit-image/scikit-image/issues/4721>`_.
+
+3. Linux on 64 bit arm (NVidia Jetson):
+   - Follow the conversation on `Issue 4705 <https://github.com/scikit-image/scikit-image/issues/4705>`_.
+
+While we do not directly support the platforms above, many of the core
+developers have experience using the platforms listed above. Do not hesitate to
+ask us questions pertaining to your specific use case.
+
+.. note::
+
+    If you would like to help package scikit-image for the platforms above, reach
+    out on github so that we may help you get started.
+
+
+.. tip::
+
+    If you need to install ``scikit-image`` one these platforms today, we hope
+    the developer instructions below can help get you started.
+
+
+End of instructions
+===================
+
+
+Installing scikit-image
+=======================
```

Comment:
Either remove the section below, or renaming it to "installing from source" or whatever...